libp2p/transports/quictransport.nim

```diff
@@ -283,22 +283,19 @@ method accept*(
 .} =
   doAssert not self.listener.isNil, "call start() before calling accept()"
 
-  while true:
-    if not self.running:
-      # stop accept only when transport is stopped (not when error occurs)
-      raise newException(QuicTransportAcceptStopped, "Quic transport stopped")
+  if not self.running:
+    # stop accept only when transport is stopped (not when error occurs)
+    raise newException(QuicTransportAcceptStopped, "Quic transport stopped")
```

Comment:
is this exception ever going to be raised?

Reply:
` doAssert not self.listener.isNil` will happen before `if not self.running:`

technically, both will not be satisfied simultaneously, unless it is a race condition (which is vaaaary low probability). but code anyway has a lot of those.